rev_news/drafts/edition-14.md

```diff
@@ -117,65 +116,65 @@ He is the maintainer of the Kernel '-stable' branches and of many
 subsystems like USB.
 
 He said that the Linux Kernel is made of more than 21 million lines of
-code in more than 53 000 files. Everything is in the tree. Driver are
-one third of the size. Nearly 4000 developers and around 400 companies
+code, in more than 53 000 files. Everything is in the tree, and drivers account
+for around one third of the size. Nearly 4000 developers and around 400 companies
 are involved.
 
 This makes the Linux Kernel the largest software project ever.
 
-Around 10 000 lines are added, 5300 lines are removed and 1800 lines
+Around 10 000 lines are added, 5300 lines are removed, and 1800 lines
 are modified, everyday!
 
 That's on average 7.8 changes per hour accross the whole tree with 5%
-in the core, 10% in the networking subsystem and 55% in the drivers.
+in the core, 10% in the networking subsystem, and 55% in the drivers.
 
-This goes against any previously thought methodology for stable
-software. And things are going faster and faster.
+This goes against all previous methodologies for stable
```

Comment:
About this line, I'd rather keep what I wrote as I think it is very close to what Greg said.